torch/_tensor.py

```diff
@@ -1661,7 +1661,7 @@ def __torch_function__(cls, func, types, args=(), kwargs=None):
 
     __torch_dispatch__ = _C._disabled_torch_dispatch_impl
 
-    def __dlpack__(self, stream=None):
+    def __dlpack__(self, stream=None, max_version=None):
```

Comment:
Note that there are also new `dl_device` and `copy` keywords ([API docs](https://data-apis.org/array-api/latest/API_specification/generated/array_api.array.__dlpack__.html)).